Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx

```diff
@@ -1173,3 +1173,19 @@ void vtkSlicerVolumeRenderingLogic::RemovePreset(vtkMRMLVolumePropertyNode* pres
     }
   presetScene->RemoveNode(preset);
 }
+
+//---------------------------------------------------------------------------
+int vtkSlicerVolumeRenderingLogic::LoadCustomPresetsScene(const char* sceneFilePath)
+{
+  if (!this->PresetsScene)
+    {
+    this->PresetsScene = vtkMRMLScene::New();
```

Comment:
This would break the GUI: all the widgets that are already created would not be updated anymore.

Reply:
I don't understand. Creating a new scene only happens if the scene was NULL before.

Reply:
OK, good then.